bindata/assets/alerts/cpu-utilization-sno.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: cpu-utilization
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-cpu-utilization
+      rules:
+        - alert: ExtremelyHighIndividualControlPlaneCPU
+          annotations:
+            summary: >-
+              CPU utilization across plane pods is high; a control plane pods outage may cause a cascading failure;
+              check control plane logs for possible issues and/or add more CPU.
+            runbook_url: PASS
+            description: >-
+              Usually, when control plane pods start to utilize more CPU, this means that there are might be
+              a issue at them. It's recommended to check pods and their logs to verify that there are not issues.
+              To fix this, increase the CPU and memory on your control plane nodes.
+          expr: | 
+            sum(
+              rate(
+                container_cpu_system_seconds_total{namespace=~"openshift-etcd|openshift-kube-apiserver|openshift-kube-controller-manager|openshift-kube-scheduler|openshift-apiserver|openshift-controller-manager|openshift-authentication",image!=""
+                }[5m]
+              )
+            ) / sum(machine_cpu_cores) * 100 
```

Comment:
`controlPlaneTopology: SingleReplica` means the control plane isn't highly available, but doesn't mean that the there are no compute nodes, does it?  I expect in practice most clusters with `controlPlaneTopology: SingleReplica` have no attached compute nodes, but the change that they might makes me a bit nervous about dropping the `kube_node_role{role="master"}` entirely.

The 60% threshold is also from [these docs][1]:

> On a large and dense cluster with three control plane nodes, the CPU and memory usage will spike up when one of the nodes is stopped, rebooted, or fails. The failures can be due to unexpected issues with power, network, underlying infrastructure, or intentional cases where the cluster is restarted after shutting it down to save costs. The remaining two control plane nodes must handle the load in order to be highly available, which leads to increase in the resource usage. This is also expected during upgrades because the control plane nodes are cordoned, drained, and rebooted serially to apply the operating system updates, as well as the control plane Operators update. To avoid cascading failures, keep the overall CPU and memory resource usage on the control plane nodes to at most 60% of all available capacity to handle the resource usage spikes. Increase the CPU and memory on the control plane nodes accordingly to avoid potential downtime due to lack of resources.

and as we adjust the associated alerts to account for single-node control planes (which makes sense to me), I think we probably also want to adjust those docs, to keep the docs and alerts in sync.

[1]: https://docs.openshift.com/container-platform/4.14/scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.html#master-node-sizing_recommended-control-plane-practices